app/models/db/MetricsFilters.scala

```diff
@@ -68,16 +71,27 @@ object MetricsFilters {
 
   def metricFilters(implicit filters: MetricsFilters): DBMetric => Rep[Option[Boolean]] = metric => checkMetricsFilters(metric)
 
-  def forksFilters[T](implicit filters: MetricsFilters): ((T, DBMetric)) => Rep[Option[Boolean]] = forkAndMetric => checkMetricsFilters(forkAndMetric._2)
+  def forksFilters(implicit filters: MetricsFilters): ((ForkFilterColumns, DBMetric)) => Rep[Option[Boolean]] = forkAndMetric =>
+    checkMetricsForkingFilters(forkAndMetric)
+
+  private def checkMetricsForkingFilters(data: (ForkFilterColumns, Schema.DBMetric))(implicit filters: MetricsFilters): Rep[Option[Boolean]] = {
```

Comment:
I'd call this `checkMetricsForkFilters` personally as the rest of the code tends to just use `fork`